src/profiler.cpp

```diff
@@ -66,9 +66,21 @@ void idfx::Profiler::Init() {
 void idfx::Profiler::Show() {
   idfx::pushRegion("Profiler::Show");
 
+  double usedMemory{-1};
+
+  // follow ISO/IEC 80000
+  const int NUNITS{5};
```

Comment:
why do you need upper case here? And in any case, we want this to be defined at compile time, so
`constexpr int nUnits = 5;`

Reply:
I'm used to upper case for constants, probably a Python idiom, but I don't mind dropping it !

Reply:
upper case is usually for preprocessor variables or macros in C/C++